codegen/generate.pl

```diff
@@ -28,17 +35,40 @@ sub gen_header {
 };
 }
 
+use constant GEN_INFO_FILE => "./codegen_hashes.tmp";
```

Comment:
Not sure if you want this file to be hidden or not. If so, it might be useful to make it a dotfile (i.e. call it `.codegen_hashes.tmp`)